iris-mpc-cpu/src/execution/hawk_main.rs

```diff
@@ -857,7 +884,15 @@ impl HawkResult {
 
         let decisions = self.match_results.decisions();
 
-        let matches = decisions.iter().map(Decision::is_match).collect_vec();
+        let matches = decisions
+            .iter()
+            .map(|&d| matches!(d, UniqueInsert).not())
+            .collect_vec();
+
+        let matches_with_skip_persistence = decisions
+            .iter()
+            .map(|&d| matches!(d, UniqueInsert | UniqueInsertSkipped).not())
+            .collect_vec();
```

Comment:
I agree that this matches the GPU implementation, however, the naming (also in the GPU version) is a bit confusing overall, no?
IMO the `matches_with_skip_persistence` vector has less matches than the matches one, so its name should rather be `matches_without_skip_persistence`, or am I seeing this one wrong @eaypek-tfh?